core/trino-spi/src/main/java/io/trino/spi/connector/ColumnHandle.java

```diff
@@ -15,4 +15,7 @@
 
 public interface ColumnHandle
 {
+    int hashCode();
```

Comment:
It is a pity that it does not help much as this methods are already in.

Maybe I can write:
```
default hashCode() { throw... ; }
```
So that way it will early. That would be better than diagnosing misterious issues. In my case `INSERT ... VALUES ` was passing nicely but no data was inserted.

Reply:
> `default hashCode() { throw... ; }`

You perhaps "can", but the impl inherited from Object still applies, so it's probably useless